### PR TITLE
footer section complete, no functionality, all looks

### PR DESCRIPTION
**Pull request by taylorseager**
## Description
This branch includes the footer that will be used on all the pages.

## Related Issue
issue ticket #14 

## Motivation and Context
This is the first pull request for the footer component

## How Can This Be Tested?
Since there is no functionality, all the buttons (11) should be at the bottom of the page, in blue. When hovering over them they should change to a light blue color. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by Zpcolburn — APPROVED** [review]
👍

File: main.css [hunk 1]
```diff
@@ -1,3 +1,37 @@
 body {
-  background-color: beige;
+  background-color: black;
+}
+
```
Comment (Zpcolburn) [inline main.css]: write comment showing this is for the footer section

**Review by Zpcolburn — COMMENTED** [review]
added comment on css.

File: main.css (hunk 1 above)
Comment (taylorseager) [reply main.css]: Done!

**Review by jessefrench — APPROVED** [review]
LGTM 👍

**Review by Zpcolburn — APPROVED** [review]
👍

**Pull request merged**